steps/artifact-pull/step-info.yml

```diff
@@ -0,0 +1 @@
+maintainer: bitrise
```

Comment:
what does this do?

Reply:
Effectively, I don't know what it does, but conceptually it just denotes that this is a Step maintained by Bitrise, not by the community.

Reply:
Workflow editor displays a badge that it's officially maintained by Bittise.

Reply:
Ah gotcha. Thanks for the info 🙇‍♂️